Source/SIMPLib/Documentation/SIMPLibFilters/CoreFilters/WriteASCIIData.md

```diff
@@ -49,11 +50,13 @@ Choice of delimiter is as follows:
 ## Parameters ##
 
 | Name             | Type | Description |
-|------------------|------|------------|
-| Output Path | File Path | The output file path |
-| File Extension | String | File extension for output file(s) |
+|------------------|------|---------_---|
```

Comment:
I don't think `_` is what you meant to put here.

Reply:
Done